packages/mobile/src/firebase/firebase.ts

```diff
@@ -241,53 +241,46 @@ export async function fetchRemoteFeatureFlags(): Promise<RemoteFeatureFlags | nu
-  // https://rnfirebase.io/remote-config/usage
+  // Don't cache values so we fetch the latest every time. https://rnfirebase.io/remote-config/usage
   await remoteConfig().setConfigSettings({ minimumFetchIntervalMillis: 0 })
-  const fetchedRemotely = await remoteConfig().fetchAndActivate()
-
-  if (fetchedRemotely) {
-    const flags: FirebaseRemoteConfigTypes.ConfigValues = remoteConfig().getAll()
-    Logger.debug(TAG, `Updated feature flags: ${JSON.stringify(flags)}`)
-
-    // When adding a new feature flag there are 2 places that need updating:
-    // the RemoteFeatureFlags interface as well as the FEATURE_FLAG_DEFAULTS map
-    // FEATURE_FLAG_DEFAULTS is in featureFlagDefaults.ts
-    // RemoteFeatureFlags is in app/saga.ts
-
-    return {
-      hideVerification: flags.hideVerification.asBoolean(),
-      // these next 2 flags are a bit weird because their default is undefined or null
-      // and the default map cannot have a value of undefined or null
-      // that is why we still need to check for it before calling a method
-      // in the future it would be great to avoid using these as default values
-      showRaiseDailyLimitTarget: flags.showRaiseDailyLimitTargetV2?.asString(),
-      celoEducationUri: flags.celoEducationUri?.asString() ?? null,
-      celoEuroEnabled: flags.celoEuroEnabled.asBoolean(),
-      shortVerificationCodesEnabled: flags.shortVerificationCodesEnabled.asBoolean(),
-      inviteRewardsEnabled: flags.inviteRewardsEnabled.asBoolean(),
-      inviteRewardCusd: flags.inviteRewardCusd.asNumber(),
-      inviteRewardWeeklyLimit: flags.inviteRewardWeeklyLimit.asNumber(),
-      walletConnectV1Enabled: flags.walletConnectV1Enabled.asBoolean(),
-      walletConnectV2Enabled: flags.walletConnectV2Enabled.asBoolean(),
-      rewardsABTestThreshold: flags.rewardsABTestThreshold.asString(),
-      rewardsPercent: flags.rewardsPercent.asNumber(),
-      rewardsStartDate: flags.rewardsStartDate.asNumber(),
-      rewardsMax: flags.rewardsMax.asNumber(),
-      rewardsMin: flags.rewardsMin.asNumber(),
-      komenciUseLightProxy: flags.komenciUseLightProxy.asBoolean(),
-      komenciAllowedDeployers: flags.komenciAllowedDeployers.asString().split(','),
-      pincodeUseExpandedBlocklist: flags.pincodeUseExpandedBlocklist.asBoolean(),
-      rewardPillText: flags.rewardPillText.asString(),
-      cashInButtonExpEnabled: flags.cashInButtonExpEnabled.asBoolean(),
-      logPhoneNumberTypeEnabled: flags.logPhoneNumberTypeEnabled.asBoolean(),
-      multiTokenShowHomeBalances: flags.multiTokenShowHomeBalances.asBoolean(),
-      multiTokenUseSendFlow: flags.multiTokenUseSendFlow.asBoolean(),
-      multiTokenUseUpdatedFeed: flags.multiTokenUseUpdatedFeed.asBoolean(),
-    }
-  } else {
-    Logger.debug('No new configs were fetched from the backend.')
-    return null
+  await remoteConfig().fetchAndActivate()
+
+  const flags: FirebaseRemoteConfigTypes.ConfigValues = remoteConfig().getAll()
```

Comment:
total drive-by nit: many of these values aren't used as "flags".  E.g., "rewardsPercent" or "rewardsMax" or "rewardPillText" do not connote "flag".  I know the naming was here before this PR, but consider something else, like `configValues`.

Reply:
Renamed :)